demo/webpack-example/src/index.ts

```diff
@@ -0,0 +1,14 @@
+import { sdk } from '@embrace-io/web-sdk';
+
+const result = sdk.initSDK({
+  appID: 'abc12',
+  appVersion: 'YOUR_APP_VERSION',
+});
+
+if (!!result) {
+  console.log('Successfully initialized the Embrace SDK');
+} else {
+  console.log('Failed to initialize the Embrace SDK');
```

Comment:
Make this `console.error`?
```suggestion
  console.error('Failed to initialize the Embrace SDK');
```